Buffs/LuluR/LuluR.cs

```diff
@@ -1,28 +1,34 @@
-﻿using System;
-using LeagueSandbox.GameServer.Logic.GameObjects;
-using LeagueSandbox.GameServer.Logic.Scripting;
-using LeagueSandbox.GameServer.Logic.API;
-
-namespace LuluR
-{
-    class LuluR : BuffGameScript
-    {
-        ChampionStatModifier statMod = new ChampionStatModifier();
-
-        public void OnActivate(Unit unit, Spell ownerSpell)
-        {
-            statMod.Size.PercentBonus = statMod.Size.PercentBonus + 1;
-            unit.AddStatModifier(statMod);
-        }
-
-        public void OnDeactivate(Unit unit)
-        {
-            unit.RemoveStatModifier(statMod);
-        }
-
-        public void OnUpdate(double diff)
-        {
-            
-        }
-    }
-}
+﻿using System;
+using LeagueSandbox.GameServer.Logic.GameObjects;
+using LeagueSandbox.GameServer.Logic.Scripting;
+using LeagueSandbox.GameServer.Logic.API;
+
+namespace LuluR
+{
+    class LuluR : BuffGameScript
+    {
+        ChampionStatModifier statMod = new ChampionStatModifier();
+
+        public void OnActivate(Unit unit, Spell ownerSpell)
+        {
+            statMod.Size.PercentBonus = statMod.Size.PercentBonus + 1;
+            statMod.HealthPoints.BaseBonus = statMod.HealthPoints.BaseBonus + 150 + 150 * ownerSpell.Level;
+            unit.GetStats().CurrentHealth = unit.GetStats().CurrentHealth + 150 + 150 * ownerSpell.Level;
+            unit.AddStatModifier(statMod);
+        }
+
+        public void OnDeactivate(Unit unit)
+        {
+            unit.RemoveStatModifier(statMod);
+            if (unit.GetStats().CurrentHealth > unit.GetStats().HealthPoints.Total)
```

Comment:
This seems wrong, surely we don't want to unbuff ALL health buffs just because our buff ended?